Documentation/gettingstarted/alibabacloud-eni.rst

```diff
@@ -0,0 +1,179 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _k8s_alibabacloud_eni:
+
+*************************************************
+Setting Up Cilium in AlibabaCloud ENI Mode (beta)
+*************************************************
+
+.. note::
+
+   This is a beta feature. Please provide feedback and file a GitHub issue if you experience any problems.
```

Comment:
Nit: we usually include the disclaimer from Documentation/beta.rst, like this:

```suggestion
.. include:: ../beta.rst

.. note::
```

But this would mean breaking the single `.. note::` you currently have into two `note`s, one created from the included file, one for the mention of the limitations (We can't indent the second note without a second `.. note::` under the `.. include::` or it is considered as an argument passed to the `.. include::` directive). Not sure what's best here. I'd probably include the disclaimer from beta.rst and have two notes, as it would be more consistent and would help finding beta features by grepping on `beta.rst`, but I don't mind much and this is up to you.